### PR TITLE
[Create Screen] Added basic functionality to add tree screen 

### PR DESCRIPTION
**Pull request by jjiang17**
## Summary
- When submit button is pressed, new tree entry is added to Firestore based on the Name and ID imputed. 

### Next Steps
- Add fields for location and date planted 
- Play with styling 

## Screenshots and Tests
IMG_3939

- if your feature is visual, attach before/after screenshots and ensure that tests still pass
CC: @mylesdomingo

### CODE REVIEW

File: src/components/ViewContainer.tsx [hunk 1]
```diff
@@ -3,8 +3,8 @@ import { StyleSheet, View } from 'react-native';
 
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
-    alignItems: 'center',
+    //flex: 1,
+    //alignItems: 'center',
```
Comment (kennethlien) [inline src/components/ViewContainer.tsx:7]: Uncomment these and style your components accordingly (I made this change, sorry).

File: src/database/firebase.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ import { Tree, Comment, Additional } from '@types';
 const config = {
   apiKey: API_KEY,
   authDomain: AUTH_DOMAIN,
-  projectId: PROJECT_ID,
+  projectId: 'bp-alemany',
```
Comment (kennethlien) [inline src/database/firebase.tsx:16]: Replace `'bp-alemany'` with `PROJECT_ID`

File: src/screens/TreeScreen.tsx [hunk 3]
```diff
@@ -1,9 +1,11 @@
 import * as React from 'react';
```
Comment (kennethlien) [inline src/screens/TreeScreen.tsx:1]: Delete this file for now, not part of task scope and made redundant by `SearchScreen`.
(Make sure to remove it from navigation as well).

File: src/screens/ViewScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
```
Comment (kennethlien) [inline src/screens/ViewScreen.tsx:1]: See GitHub Actions comment

**Review by kennethlien — DISMISSED** [review]

File: src/components/ViewContainer.tsx [hunk 5]
```diff
@@ -4,9 +4,9 @@ import { StyleSheet, View } from 'react-native';
 
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
-    alignItems: 'center',
     justifyContent: 'center',
```
Comment (mylesdomingo) [inline src/components/ViewContainer.tsx:7]: I would revert this file changes.

File: src/screens/AddScreen.tsx [hunk 6]
```diff
@@ -13,14 +15,36 @@ const styles = StyleSheet.create({
...
+    planted: null,
+  });
+
   return (
     <ViewContainer>
       <Title>Create Screen</Title>
       <View style={styles.separator} />
-      <EditScreenInfo path="/screens/AddScreen.tsx" />
+      <TextInput
+        label="Name"
```
Comment (mylesdomingo) [inline src/screens/AddScreen.tsx:36]: I think using tree instead of entry is fine here

**Review by mylesdomingo — APPROVED** [review]

**Pull request merged**